data/tlg0020/tlg001/tlg0020.tlg001.perseus-eng2.xml

```diff
@@ -30,14 +36,20 @@
             <biblStruct>
                <monogr>
                   <author>Hesiod</author>
-                  <title>The Homeric Hymns and Homerica with an English Translation by
-                                   Hugh G. Evelyn-White. Theogony.</title>
+                  <title>The Homeric Hymns and Homerica</title>
+                    <editor role="translator">Hugh G. Evelyn-White</editor>
                   <imprint>
-                     <publisher>Cambridge, MA.,Harvard University Press; London,
-                                        William Heinemann Ltd.</publisher>
+                      <pubPlace>London</pubPlace>
+                     <publisher>William Heinemann Ltd.</publisher>
+                       <pubPlace>New York</pubPlace>
+                       <publisher>The Macmillan Co.</publisher>
                      <date>1914</date>
                   </imprint>
                </monogr>
+                 <series>
+                      <title>Loeb Classical Library</title>
+                 </series>
+                 <ref target="https://archive.org/details/hesiodhomerichym00hesiuoft/hesiodhomerichym00hesiuoft/page/78/mode/2up">Internet Archive</ref>
```

Comment:
@AlisonBabeu There are other irregularities in this header (typo in refsDecl, incomplete change log, etc) so the missing schema is a clue that this was not reviewed for deeper issues with content.

Reply:
hi @lcerrato yeah I started to realized the depth of the challenges as I went through but was focused on this first pass with fixing obvious stuff, since they were already showing up in Scaife with almost no metadata.  I will separate out the Hesiod into a separate pull request I think as you suggest.

Reply:
@AlisonBabeu I can see other stuff like a misplaced note and an `<extent>` tag. Looks like these were just one of the auto conversion batches. The problem is I can't see the whole file in this format, only the parts you changed. Otherwise, I would make changes here.

Reply:
hi @lcerrato so do we no longer include extent tags and where should the note go just 
out of curiosity?

Reply:
No, I have been eliminating these.
I started moving the information seen above to the change logs (which are entirely commented out in this file).

I think it would be best to leave as is and I will revisit as part of larger review.

Reply:
Not recommending this is a priority, but it would look like:

`<note anchored="true" place="unspecified">Text keyboarded in the Philippines.  Basic SGML tagging by Scott Ettinger.</note>`

could be something like 
`<change who="Scott Ettinger" notAfter="1991-07">Basic SGML tagging.</change>`
`<change notAfter="1991-07">Text keyboarded in the Philippines.</change>`

Reply:
hi @lcerrato I will finish off the bibliographic changes etc. for Hesiod and then do a separate pull request for the filesof the Homeric Hymns without worrying about the larger header issues for the moment then.